src/components/Policy/TableToolbar/PolicyTableToolbar.tsx

```diff
@@ -62,21 +61,19 @@ const filterMetadata: ColumnsMetada<typeof PolicyFilterColumn> = {
         label: 'Enabled',
         placeholder: 'Filter by enabled status',
         options: {
-            default: 'all',
-            exclude: [ 'all' ],
-            exclusive: true,
+            exclusive: false,
             items: [
                 {
                     value: 'all',
                     label: <>All</>
```

Comment:
Is this one needed?

Reply:
Nope! I read part of the issue wrong, removing now!